Dockerfile.redhat

```diff
@@ -118,17 +118,10 @@ RUN mv .deps/install/lib64 .deps/install/lib
 
 ####### End of AWS S3 SDK
 
-# Build OpenCV from source (for binary inputs and custom nodes)
-ARG OPENCV_BRANCH="4.6.0"
-WORKDIR /opt
-RUN git clone https://github.com/opencv/opencv.git --depth 1 -b ${OPENCV_BRANCH} /opt/opencv_repo
-COPY src/opencv_cmake_flags.txt /opt/opencv_repo
-WORKDIR /opt/opencv_repo/build
-# hadolint ignore=SC2046
-RUN cmake $(cat /opt/opencv_repo/opencv_cmake_flags.txt) /opt/opencv_repo && \
-    make "-j$(nproc)" && \
-    make install
-# End of OpenCV building
+####### Build OpenCV
+WORKDIR /ovms/third_party/opencv
+RUN chmod +x install_opencv.sh && ./install_opencv.sh
```

Comment:
this script already has executable permissions. chmod is not needed

Reply:
fixed